static/bidder-params/yieldlab.json

```diff
@@ -27,7 +23,6 @@
   },
   "required": [
     "adslotId",
-    "supplyId",
-    "adSize"
```

Comment:
Removing a required parameter can break publishers. We recommend making this an optional parameter to start with and then work with publishers using the Yieldlab adapter to remove this parameter from their configuration and then once it's certain that nobody is using this parameter anymore, it can be removed from this yaml file